builtins.go

```diff
@@ -898,6 +952,20 @@ func builtinStrReplace(i *interpreter, trace traceElement, strv, fromv, tov valu
 	return makeValueString(strings.Replace(sStr, sFrom, sTo, -1)), nil
 }
 
+func buildinBase64Decode(i *interpreter, trace traceElement, strv value) (value, error) {
+	str, err := i.getString(strv, trace)
+	if err != nil {
+		return nil, err
+	}
+
+	sDec, err := base64.StdEncoding.DecodeString(string(str.value))
+	if err != nil {
+		return nil,  i.Error(fmt.Sprintf("failed to decode string: %s", err), trace)
+	}
+
+	return makeValueString(string(sDec)), nil
```

Comment:
Here you implictly assume UTF-8 encoding. Sadly, for compatibility reasons we need to use "naive" encoding instead (codepoint = byte, all codepoints must be <=255). Because of that this function is pretty much deprecated in favor of `std.base64DecodeBytes` which requires an explicit choice of string encoding.

Reply:
ok. I'll keep working on base64DecodeBytes, and I'll update this to use that method once it's ready.